network/common/pom.xml

```diff
@@ -55,6 +55,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
+      <scope>compile</scope>
```

Comment:
> Isn't compile the default scope? i.e. couldn't you leave out the explicit compile scope here and just not specify a scope?



> Isn't compile the default scope? i.e. couldn't you leave out the explicit compile scope here and just not specify a scope?

It's set to `provided` in the parent pom.xml: https://github.com/apache/spark/blob/b23c4521f5df905e4fe4d79dd5b670286e2697f7/pom.xml#L365